doc/apiref.md

```diff
@@ -64,7 +64,7 @@ master.
     stage_env_manifests_path = get_manifests_path(master, :env => 'staging')
     ```
 
-### `get_site_pp_path`
+### `get_site_pp_path(host)`
```

Comment:
The options hash is only documented in the examples. It would be nice to have a full description of what options are available.